docs/running-a-node/security.md

```diff
@@ -0,0 +1,50 @@
+title: Node Security | Nano Documentation
+description: Information about how to properly secure your Nano node when getting setup on the network.
+
+There are [many reasons to run a Nano node](/running-a-node/overview/#why-run-a-node) on the network. Nodes are the participants that help vote on transaction validity, assist other nodes with bootstrapping blocks in the ledger and providing an access point to all accounts. But those who choose to run them should be making a long-term commitment to [run them on proper hardware](/running-a-node/node-setup/#hardware-recommendations), keep them updated with the [latest release](/releases/node-releases/) and, most importantly, keep their setup as secure as possible.
+
+The details below are guidelines on things to watch out for when setting up and securing your Nano node. As the node can be run on many different operating systems, some of these guidelines have been kept more general. There are plenty of resources online for learning how to apply these guidelines to more specific setups and additional details will be included in the docs here as they are appropriate.
+
+## Node configuration
+
+### Enabling control
+
+Various RPC calls are marked as requiring the `enable_control` option to be turned on before they can be called. This extra level of permission to use these RPCs was put in place because the calls can be dangerous in a couple ways:
+
+* They can potentially allow access to wallet funds
+* They can consume extra node resources compared to other calls, such as using more disk space or requiring additional computation to complete
+
+By turning `enable_control` on, anyone with access to your RPC can run these potentially dangerous commands, so it is only recommended with [port configurations](#port-configuration) where RPC access is restricted to local and loopback addresses only. If your RPC is exposed to external or non-loopback addresses, the node will print out warnings to `stdout` and your logs to help make you aware of potential exposure.
+
+### Port configuration
+
+Opening default port `7075` on UDP and TCP is required for the node to participate on the main network and this should be done unrestricted. The default port for RPC access is `7076` and should only be available to those you wish to have control of the node. Verifying the configuration in `config-rpc.toml` file for `node.address` and `node.enable_control` should be done on all nodes, alonside other access verifications outlined below.
```

Comment:
Similarly `address` (and `enable_control`) are in the top level for `config-rpc.toml`